_posts/en/newsletters/2020-06-17-newsletter.md

```diff
@@ -0,0 +1,193 @@
+  improving both participant privacy and onchain scalability.  In that
+  sense, the protocol acts as a sort of generalized [channel
+  factory][topic channel factories] that applies not just to LN but to
+  many protocols that create onchain transactions with unique
+  fingerprints.
+
+    The authors outline how CoinPool could work using existing features
+    of Bitcoin plus taproot, [SIGHASH_NOINPUT][topic sighash_noinput], and the ability to use
+    a delete-only accumulator via Bitcoin Script (e.g. a merkle tree;
+    perhaps something like the proposed [BIP116][]
+    `OP_MERKLEBRANCHVERIFY`).  They don't appear to be advocating a
+    specific design but instead want to start a discussion about how
+    CoinPool or something like it could provide a generalized mechanism
+    that wallets use by default to eliminate the onchain footprint of
+    various current and proposed multi-user protocols.
+
+- **WabiSabi coordinated coinjoins with arbitrary output values:** in
+  the [coinjoin][topic coinjoin] protocol, a group of users
+  collaboratively create a transaction template that spends some of
+  their existing UTXOs (inputs) to a new set of UTXOs (outputs).  The
+  way this transaction template is created has implications for the
+  privacy provided by the coinjoin, and different implementations
+  use different methods:
+
+    <!-- Taker creates tx template, see discussion between harding and
+    waxwing: http://gnusha.org/joinmarket/2020-06-14.log -->
+
+    - [Joinmarket][] has the user paying for the coinjoin (the *market
+      taker*) contact each of the users being paid for providing BTC
+      liquidity (the *market makers*), collect their input and output
+      information, and create the transaction template.  This gives the
+      taker knowledge of which inputs fund which outputs for all
+      participants in the coinjoin, but it also ensures that each maker
+      only has knowledge about which of their inputs funds which of
+      their outputs.  The taker directly gains the privacy benefits of
+      the coinjoin and the makers directly gain income for providing
+      liquidity.  If the taker preserves his own privacy, the makers
+      also indirectly gain increased privacy against third party block
+      chain surveillance.  Makers who want guarantees about their privacy
+      can always operate as takers for a few rounds of mixing.
+
+        <!-- Quotes from joinmarket README.md:
+          - "Ability to spend directly, or with coinjoin"
+          - "Can specify exact amount of coinjoin (figures from 0.01 to 30.0 btc"
+        -->
+
+        This model gives takers a lot of flexibility with their own
+        inputs and outputs to the transaction template.  For example, a
+        taker can choose the amounts of the coinjoin they want to create
+        or can spend their money to a third party as part of a coinjoin.
+
+    - [Wasabi][] uses a centralized coordinator who organizes every
```

Comment:
The coordinator is a "who" in this sentence and an "it" further down in the same paragraph.